### PR TITLE
886555-UG documentation for the digital gauge control Images

### PR DESCRIPTION
**Pull request by SriRadheshNagS**
Added Images for the digital gauge ug documentation

### CODE REVIEW

**Review by saiganesh-sakthivel — APPROVED** [review]

File: MAUI/DigitalGauge/Character-Display-Types.md [hunk 1]
```diff
@@ -34,6 +34,9 @@ Numbers can be displayed in a digital gauge in different formats using the `Char
 
 {% endtabs %}
 
+![Display Number](Images\DigitalGauge9.png)
```
Comment (Jeyasri-Murugan) [inline MAUI/DigitalGauge/Character-Display-Types.md:37]: DigitalGauge9. - name properly related feature for SEO

Comment (saiganesh-sakthivel) [inline MAUI/DigitalGauge/Character-Display-Types.md:37]: add name in small letter

display-number

Comment (saiganesh-sakthivel) [inline MAUI/DigitalGauge/Character-Display-Types.md:37]: reference like this

[tooltip-template-in-maui-stepprogressbar]

Comment (SriRadheshNagS) [reply MAUI/DigitalGauge/Character-Display-Types.md]: Updated Changes

Comment (SriRadheshNagS) [reply MAUI/DigitalGauge/Character-Display-Types.md]: Updated Changes

Comment (SriRadheshNagS) [reply MAUI/DigitalGauge/Character-Display-Types.md]: Updated Changes

**Review by saiganesh-sakthivel — APPROVED** [review]
fine

**Pull request merged**